docs/endpoints/states.md

```diff
@@ -87,42 +87,41 @@ Returns the data for a single `:state` state.
 `GET https://api.corona-zahlen.org/states/HH`
```

Comment:
Eventuell sollten wir die URL updaten und wenn wir schon dabei sind, die baseURL zentral in einer variablen speichern.
Gilt für jedes mal, wenn die "alte url" (api.corona-zahlen.org) gehardcodet ist